lib/Gateway.js

```diff
@@ -1080,8 +1103,11 @@ Gateway.prototype.rediscoverNode = function (nodeID) {
     nodeDevices.forEach(device => this.discoverDevice(node, device))
 
     // discover node values (that are not part of a device)
-    for (const id in node.values) {
-      this.discoverValue(node, id)
+    // iterate prioritized first, then the remaining
+    for (const groupedValues of getPriorityCCFirst(node.values)) {
+      for (const id of groupedValues) {
```

Comment:
I don't understand this, you should just use 

node.values[vId] here, you don't need another nested for

Reply:
it is simplified once we used the single array